Misc/NEWS.d/next/Core and Builtins/2017-09-26-13-03-16.bpo-31588.wT9Iy7.rst

```diff
@@ -0,0 +1,2 @@
+Raise a `TypeError` with a helpful error message in case class creation failed
+due to a metaclass with a bad ``__prepare__()`` method. Patch by Oren Milman.
```

Comment:
Should I be more specific here?
i.e. it could be:
... due to a metaclass whose ``__prepare__()`` method returned a non-mapping object.

Reply:
I think your current wording is fine.